### PR TITLE
refactor: send_batched_transactions optimisation

### PR DESCRIPTION
**Pull request by sergeytimoshin**

### CODE REVIEW

File: forester/src/epoch_manager.rs [hunk 1]
```diff
@@ -728,10 +723,10 @@ impl<R: RpcConnection, I: Indexer<R>> EpochManager<R, I> {
                 let batch_tx_future = send_batched_transactions(
                     &self.config.payer_keypair,
                     self.rpc_pool.clone(),
-                    config, // TODO: define config in epoch manager
+                    &config, // TODO: define config in epoch manager
                     tree.tree_accounts,
                     &transaction_builder,
-                    epoch_pda.epoch,
+                    // epoch_pda.epoch,
```
Comment (ananas-block) [inline forester/src/epoch_manager.rs:729]: leftover?

File: forester/src/send_transaction.rs [hunk 2]
```diff
@@ -126,52 +128,72 @@ pub async fn send_batched_transactions<T: TransactionBuilder, R: RpcConnection>(
...
-                for transaction in transactions.iter() {
-                    let res = send_signed_transaction(transaction, &rpc, config.retry_config);
-                    results.push(res);
-                }
-                let all_results = join_all(results);
-                all_results.await;
+
+            let batch_start = Instant::now();
+            let url = rpc.get_url().to_string();
+            let remaining_time = config
```
Comment (ananas-block) [inline forester/src/send_transaction.rs:138]: Nice!

**Review by ananas-block — APPROVED** [review]
Nice optimization. Just saw the one leftover comment.

File: forester/src/epoch_manager.rs (hunk 1 above)
Comment (sergeytimoshin) [reply forester/src/epoch_manager.rs]: yes, fixed

**Pull request merged**